topic05-git-advanced/book/02.02.md

```diff
@@ -0,0 +1,85 @@
+# Rewriting History
+
+## Prerequisites
+
+* Git (2.x)
+* (Optional) hub - https://github.com/github/hub
+* Access to the fheng/git-tutorial repo (ask if you don't have)
+
+Clone this repository
+
+```bash
+# Don't fork
+git clone git@github.com:fheng/git-tutorial.git
```

Comment:
This is a private repo so it won't be possible. I think we should make a copy of it in the feedhenry org, with just the required branches.